Backend/app/rider-platform/rider-app/Main/spec/Storage/frfs-ticket.yaml

```diff
@@ -205,6 +208,16 @@ FRFSTicketBooking:
       kvFunction: updateWithKV
       params: [bppBankAccountNumber, bppBankCode, updatedAt]
       where: id
+    updateRefundAndCancellationChargesAndIsCancellableByBookingId:
```

Comment:
`And` can be in the end alone
`updateRefundCancellationChargesAndIsCancellableByBookingId`